### PR TITLE
Packaging is fun

### LINKED ISSUE #313: conu is unusable in Fedora
```
pkg_resources.DistributionNotFound: The 'websocket-client==0.51.0' distribution was not found and is required by conu
```

We need to relax the requirements in setup.py to use whatever version of the direct dependency. For req.txt, we can continue doing what we're doing.

### LINKED ISSUE #314: enum strikes back
When I install conu from PyPI, this is what happens:
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/usr/lib/python3.7/site-packages/setuptools/__init__.py", line 6, in <module>
        import distutils.core
      File "/usr/lib64/python3.7/distutils/core.py", line 16, in <module>
        from distutils.dist import Distribution
      File "/usr/lib64/python3.7/distutils/dist.py", line 9, in <module>
        import re
      File "/usr/lib64/python3.7/re.py", line 143, in <module>
        class RegexFlag(enum.IntFlag):
    AttributeError: module 'enum' has no attribute 'IntFlag'
```

Do we just create a new upstream release?

### PR DESCRIPTION
**Pull request by TomasTomecek**
And let's ship this after it's merged!

Fixes #314
Fixes #313

### CODE REVIEW

**Comment by TomasTomecek** [comment]
just fixed the only codacy nit; should be nice and green

File: setup.py [hunk 1]
```diff
@@ -32,11 +32,15 @@
...
+    with open("./requirements.in") as fd:
         for line in fd.readlines():
             if not line or line.startswith('-i'):
                 continue
-            packages.append(line.split(';', 1)[0])
+            parsed_line = line.split(';', 1)[0].strip()
+            packages.append(parsed_line)
+
+    if sys.version_info.major == 3:
+        packages.remove("enum34")
```
Comment (jpopelka) [inline setup.py]: Not sure why is this needed, but OK. Can you perhaps add a comment (into setup.py) explaining the hack ?
